### DISCUSSION TITLE
Full screen start menu

### DISCUSSION THREAD

**Discussion by thatnokiaguy** [valinet/ExplorerPatcher#951] [Ideas]
wondering if there are any plans to make the windows 10 menu option have a fullscrrne option?

Thanks

**Comment by valinet** [comment]
Implemented in latest pre-release (42.23).

**Comment by rcamarda390** [comment]
I am checking properties for full screen start, and I am unable to find it.
Wait, I found Start Menu > Start menu Style: (change to Windows 10).
all my windows 10 tiles, groupings are back!!!!
this was the _only_ reason why I didnt want to upgrade to 11!
TY!
